ikarus/python/finiteelements/registerpreelement.hh

```diff
@@ -19,7 +19,7 @@ namespace Ikarus::Python {
 template <class NonLinearElasticPre, class... options>
 void registerNonLinearElasticPre(pybind11::handle scope, pybind11::class_<NonLinearElasticPre, options...> cls) {
   using Material = typename NonLinearElasticPre::Material;
-  cls.def(pybind11::init([](const Material& mat) { return new NonLinearElasticPre(mat); }));
+  cls.def(pybind11::init([](const Material& mat, double density) { return new NonLinearElasticPre(mat, density); }));
```

Comment:
It might also be reasonable to add density to the material type. For instance I think muesli also wants the density for its MaterialPoint.

Reply:
Yeah I actually thought about it as well. Shall I change it? Maybe this clashes then with the uhyperelastic material PRs